src/main/java/games/strategy/triplea/ui/menubar/LobbyMenu.java

```diff
@@ -284,7 +289,7 @@ private void addUnbanMacAddressMenu(final JMenu parentMenu) {
     parentMenu.add(item);
   }
 
-  private long requestTimespanSupplication() {
+  private TemporalAmount requestTimespanSupplication() {
```

Comment:
This method name has been driving me nuts since I first saw it a couple of months ago.  What does "supplication" even mean in this context?

Can you think of a better name?  I'll suggest `requestBanDuration` or `promptForBanDuration`, but there's probably something better.